src/find-user.ts

```diff
@@ -0,0 +1,22 @@
+import { User } from './entity/User';
+import { AppDataSource } from './data-source';
+import { CustomError } from './custom-errror';
+
+export async function findingUser(email: string) {
+  const firstUser = await AppDataSource.getRepository(User)
+    .createQueryBuilder('user')
+    .where('user.email = :email', { email: email })
```

Comment:
curiosidade: quando voce quer passar uma variável com o mesmo nome do campo em JS/TS, você nao precisa repetir o nome

```suggestion
    .where('user.email = :email', { email })
```

eu acho que isso funciona :D